package.json

```diff
@@ -25,7 +25,7 @@
     "test": "npm run test:version && npm run lint && npm run test:cov",
     "test:version": "node scripts/check-version.js",
     "test:cov": "c8 npm run test:test",
-    "test:test": "mocha",
+    "test:test": "mocha --reporter min",
```

Comment:
Can we put this in .mocharc.json in the project root along with other mocha config settings?

Reply:
yes we can 😄